source/ios/AdaptiveCards/ADCIOSFramework/ADCIOSFramework/ACRSeparator.h

```diff
@@ -0,0 +1,24 @@
+//
+//  ACRSeparator.h
+//  ADCIOSFramework
+//
+//  Copyright © 2017 Microsoft. All rights reserved.
+//
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+#import "HostConfig.h"
+#import "SharedAdaptiveCard.h"
+
+@interface ACRSeparator:UIView
+
+@property CGFloat dimension_width;
+@property CGFloat dimension_height;
+@property CGFloat lineWidth;
+@property UILayoutConstraintAxis axis;
+@property long    rgb;
+
++ (void) renderSeparation:(std::shared_ptr<AdaptiveCards::BaseCardElement> const &) elem
+             ForSuperView:(UIView* ) view
+           withHostConfig:(std::shared_ptr<AdaptiveCards::HostConfig> const &) config;
+
+@end
```

Comment:
Please try to maintain a consistent code style:
* lowercase f in forSuperview (also lowercase v to match Apple's style)
* camelCasing instead of underscores in dimension_width and dimension_height
* consistent spacing around "ACRSeparator:UIView", "long     rgb", and "(UIView* )"